job_spec/WATER_MAP_EQ.yml

```
@@ -0,0 +1,114 @@
+WATER_MAP_EQ:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default: '""'
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 1
+        items:
+          anyOf:
+            - description: The name of the IW VV+VH Sentinel-1 GRDH granule to process
+              type: string
+              pattern: "^S1[AB]_IW_GRDH_1SDV"
```

```suggestion
              pattern: "^S1[AB]_IW_GRDH_1S[SD]V"
```
Fritz's algorithm only looks at the VV polarization, so `SV` or `DV` granules should work